qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -273,6 +273,25 @@ def convert_set_frequency(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(instructions.ShiftFrequency)
+    def convert_shift_frequency(self, shift, instruction):
+        """Return converted `ShiftFrequency`.
+
+        Args:
+            shift (int): Offset time.
+            instruction (ShiftFrequency): Shift frequency instruction.
+
+        Returns:
+            dict: Dictionary of required parameters.
+        """
+        command_dict = {
+            'name': 'shiftf',
+            't0': shift+instruction.start_time,
+            'ch': instruction.channel.name,
+            'frequency': instruction.frequency / 1e9
+        }
+        return self._qobj_model(**command_dict)
```

Comment:
Why not this? It looks more clear to me.. 
```suggestion
        return self._qobj_model(name='shiftf',
                               t0=shift+instruction.start_time,
                               ch=instruction.channel.name,
                               frequency=instruction.frequency / 1e9)
```

Reply:
Oh.. I see. It's the same style was [there before](https://github.com/Qiskit/qiskit-terra/pull/4390/files#diff-78f2c68464fc91f622184a3afeaf0a8bR268-R274). I appreciate the consistency more that I dislike the approach. Maybe for another time.

Reply:
Yes, I simply used it for consistency